nominees/emergency-relief-package-tracker.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Emergency Relief Package Tracker",
+  "description": "This project is designed to help local organizations and government institutions help track distribution of emergency relief packages along with their contents to ensure essential supplies are evenly distributed.",
+  "website": "https://github.com/africa-covid-19-response-toolkit/emergency-relief-package-tracker",
```

Comment:
@318anushka kindly update the website field to point to a websiteURL rather than Github repo. Thanks.